### DISCUSSION TITLE
Sandbox Mode

### DISCUSSION THREAD

**Discussion by Cat-o-War** [redeclipse/base#1598] [Ideas and Feature Requests]
Red Eclipse 2.1 does not have a sandbox mode (bots do not do anything), unlike RE 2.0. The mode is quite useful for new players, as they can test out the weapons. So I am requesting a sandbox mode, like the one in RE 2.0.

**Comment by qreeves** [comment]
I am not sure what the sandbox mode is that you speak of. Bot behaviour like this has always been controlled with the `botabilities` variable.
